.rubocop.yml

```diff
@@ -1,18 +1,314 @@
-inherit_mode:
-  merge:
-    - Exclude
-
 require:
-  - standard
-  - standard-custom
-  - standard-performance
-  - rubocop-performance
```

Comment:
Did you find this require was no longer necessary? https://github.com/rubocop/rubocop-performance#rubocop-configuration-file seems to suggest we want this here but maybe we have some other ✨ wired up.

Reply:
My mistake. Fixed.